src/TweetPublisher/TwitterApi/TwitterApiWrapper.php

```diff
@@ -50,28 +51,54 @@ public function __construct(
     }
 
     /**
-     * @return mixed[]
+     * @return string[]
      */
     public function getPublishedTweets(): array
     {
-        $fullTweets =  $this->callGet(self::TIMELINE_URL, '* from:' . $this->twitterName, [
-            'count' => 70, // these will be filtered down by following conditions; at least number of posts
-            'trim_user' => true, // we don't need any user info
-            'exclude_replies' => true, // we don't need replies
-            'include_rts' => false, // we don't need retweets
-            'since_id' => 824225319879987203 // this started at 2017-08-20, nothing before
-        ]);
+        $rawTweets = $this->getPublishedTweetsRaw();
 
-        $fullTweets = $this->tweetEntityCompleter->completeOriginalUrlsToText($fullTweets);
+        $rawTweets = $this->tweetEntityCompleter->completeOriginalUrlsToText($rawTweets);
 
         $tweets = [];
-        foreach ($fullTweets as $fullTweet) {
+        foreach ($rawTweets as $fullTweet) {
             $tweets[] = $fullTweet['text'];
         }
 
         return $tweets;
     }
 
+    public function publishTweet(string $status): void
+    {
+        $this->callPost(self::UPDATE_URL, [
+            'status' => $status
+        ]);
+    }
+
+    public function getDaysSinceLastTweet(): int
+    {
+        $rawTweets = $this->getPublishedTweetsRaw();
+        $lastRawTweet = array_pop($rawTweets);
+
+        $tweetPublishDate = DateTime::from($lastRawTweet['created_at']);
+        $dateDiff = $tweetPublishDate->diff(new DateTime('today'));
+
+        return (int) $dateDiff->format('%a');
+    }
+
+    /**
+     * @return mixed[]
+     */
+    private function getPublishedTweetsRaw(): array
+    {
+        return $this->callGet(self::TIMELINE_URL, '* from:' . $this->twitterName, [
```

Comment:
Máš pravdu. Kdyby to mělo na něco vliv, tak bych to cachoval. Tj. až mi Twitter řekne, že mám moc volání za den. Teď to vychází na 3-4 volání API denně.